### ISSUE TITLE
have the encryption icon toggle

### ISSUE THREAD

**Issue by tobibeer** [danielo515/TW5-EncryptTiddlerPlugin#8]
this is just candy, would be nice if the icon reflected the current tiddler's state

closed lock => encrypted
open lock => not encrypted

**Comment by danielo515** [comment]
Hello Tobias 
That was my initial intention, but then I thought about the camouflage feature. After one tiddler is encrypted you can change it's appearance so it doesn't look like an encrypted tiddler. If I provide that visual feedback I will totally kill this feature. 

Regards

**Comment by kookma** [comment]
@danielo515 

There is several drawback with this feature right now

1. The tiddler content display the tiddler is encrypted
2. The encrypt filed shows the tiddler is encrypted
3. How the author of wiki can realize if a tiddler is encrypted or not?

In my opinion the padlock should change from open to close! If not it is better to use a setting tab to encrypt decrypt and do not use padlock icon.

**Comment by danielo515** [comment]
I think that most users rely on the padlock to change, and almost nobody takes advantage of the fact that it does not. I'll change that then.

**Comment by hchaase** [comment]
while your at it, could you make it possible to toggle the visibility of the padlock, like with the other buttons in the tiddler (from info > tools). This way, you could take the padlock out of sight, and kind of hide the encryption status, as original intended. 

this would also make it more concistent with the rest of TW

**Comment by danielo515** [comment]
> while your at it, could you make it possible to toggle the visibility of the padlock, like with the other buttons in the tiddler (from info > tools). This way, you could take the padlock out of sight, and kind of hide the encryption status, as original intended.
> 
> this would also make it more concistent with the rest of TW

Yes, I will. But note that the pop-up will probably not work on that scenario. 
The reason this is not possible now is because this plugin is old, and I was just a young unexperienced boy testing this proof of concept 😁
